tests/_serialization/test_taurus_interop.py

```diff
@@ -15,7 +15,7 @@ def test_flatten():
     bounds = CategoricalBounds(categories=["foo", "bar"])
     template = ProcessTemplate(
         "spam",
-        conditions=[(ConditionTemplate(name="eggs", bounds=bounds), bounds)]
+        conditions=[[ConditionTemplate(name="eggs", bounds=bounds), bounds]]
```

Comment:
What happens if this is a tuple instead of a list?

Reply:
good catch, it's a leftover debug thing, putting back